src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -178,7 +178,7 @@ const UsersFragment = ({
             image: item.user ? item.user.profile_pic_path : item.profile_pic_path,
             isunfollowed: isUnfollowed,
             description: item.user ? item.user.bio : item.bio,
-            karmaScore: item.karma_score
+            karmaScore: item.user ? item.user.karma_score : item.karma_score
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one. It adds a check to ensure that `item.user` exists before trying to access its `karma_score` property. This will prevent potential runtime errors if `item.user` is `null` or `undefined`.
```diff
- karmaScore: item.karma_score
+ karmaScore: item.user ? item.user.karma_score : item.karma_score
```
This change improves the robustness of the code by handling potential null or undefined values for `item.user`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->